app/Notifications/TenantApprovalRequest.php

```
@@ -0,0 +1,76 @@
+     *
+     * @return array<int, string>
+     */
+    public function via(object $notifiable): array
+    {
+        return ['mail', 'database'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     */
+    public function toMail(object $notifiable): MailMessage
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (Unused Code Rules)

(UnusedFormalParameter)